### ISSUE TITLE
Feature request: Set max time per mission calculation

### ISSUE THREAD

**Issue by zeenk** [teelolws/TLDRMissions#147]
**TLDR Version of your idea**
TLDR version: Add a max allowed time per mission timer if u have the new option "Immediately start missions once calculated" selected

**Slightly longer version**
Now that you have added the option to immediately start missions once they are calculated (AMAZING feature btw) I was thinking if it would be possible to make this feature even better by adding a timer on how long you allow the calculations to go on for until it skips a mission? maybe add some sort of slider or just a box the user can input the the max allowed seconds per mission they want similar to the sims per frame. or would this cause to many bugs with the sims?

Zaphon had a similar idea in #132

**Comment by teelolws** [comment]
From the other issue, I'm still considering it. But have more important parts of the addon to work on at the moment. Like getting those processing times down. Its mostly spells with "randomness" that take forever to calculate, and I'm trying to find alternative ways to simulate them.

**Comment by zeenk** [comment]
Alright  👍
